tests/backend/http/test_http.py

```diff
@@ -338,10 +338,17 @@ async def test_get_static(backend_http_send):
 
 
 @pytest.mark.trio
-async def test_get_redirect(backend_http_send):
-    status, headers, body = await backend_http_send("/redirect/foo/bar?a=1&b=2")
+@pytest.mark.parametrize("kind", ["ascii", "unicode"])
+async def test_get_redirect(backend_http_send, kind):
+    if kind == "ascii":
+        target = "foo/bar?a=1&b=2"
+    else:
+        assert kind == "unicode"
+        target = "foo/bar?a=1&b=2"
```

Comment:
I don't see any unicode char (compared to the `target` defined for the `kind` `ascii`) is this expected ?

Reply:
good catch ! I've fixed the test :+1: